encrypted-media/scripts/playback-temporary-expired.js

```diff
@@ -44,16 +44,18 @@ function runTest(config,qualifier) {
 
             assert_in_array(event.messageType, ['license-request', 'individualization-request']);
 
-            var expiration = Date.now().valueOf();
-            config.messagehandler(event.messageType, event.message, expiration).then(function(response) {
-                return event.target.update(response);
-            }).then(test.step_func(function(){
-                assert_approx_equals(event.target.expiration, expiration, 2000, "expiration attribute should equal provided expiration time");
-                test.step_timeout(function() {
-                    _video.play();
-                    test.step_timeout(function() { test.done(); }, 2000);
-                }, 5000);
-            })).catch(onFailure);
+            var expiration = Date.now().valueOf() - 60000;
```

Comment:
Hmm, I was finding that the server rounds up the expiration time to the (server's) current time.

Anyway, if we do not have robust server support for obtaining an expired license, we cannot implement this test.